src/Helpers/PriceHelper.php

```diff
@@ -334,4 +359,9 @@ private function getRootPrice(?PriceList $priceList, ?Price $price): ?Price
 
         return $parentPrice ?? $price;
     }
+
+    private function fireEvent(string $event): void
```

Comment:
better use protected, gives the possibility to override the function in the project even if this is propably never the case here, doesnt hurt